core/src/banking_stage/consumer.rs

```diff
@@ -684,6 +709,39 @@ impl Consumer {
         }
     }
 
+    pub fn check_fee_payer_unlocked(
```

Comment:
A lot of lifting in this function, the saving is remove bad transactions upfront, and warms up good transactions' payer accounts, the extra time spent here is still a concern, especially if a lot of transactions going through this check. Can we add another metrics to monitor it?